src/DetailView/DetailView.js

```diff
@@ -117,7 +135,7 @@ class DetailView extends ConnectionComponent {
 
 	render() {
 		const entity = this.props.entity.value;
-		const eventTypes = this.props.eventTypes.value;
+		this.eventTypes = this.props.eventTypes.value;
```

Comment:
Was changed during development to fix a bug, is not necessary anymore, so it was changed back. Thanks for noticing!